packages/config/config/devices/0x027a/zse11.json

```
@@ -67,22 +79,160 @@
 			"label": "Flash LED on Motion",
 			"defaultValue": 1
 		},
+		{
+			"#": "20",
+			"$if": "firmwareVersion >= 2.0",
+			"label": "Vibration Sensor",
+			"description": "Vibration sensor enable or disable.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"options": [
+				{
+					"label": "Disable vibration sensor",
+					"value": 0
+				},
+				{
+					"label": "Enable vibration sensor",
+					"value": 1
+				}
+			]
+		},
```

This should just extend the `base_enable_disable` template. Also we avoid repeating the same information within a config parameter. In this case "vibration sensor" is mentioned 4 times.

```suggestion
		{
			"#": "20",
			"$if": "firmwareVersion >= 2.0",
			"$import": "~/templates/master_template.json#base_enable_disable",
			"label": "Vibration Sensor",
			"defaultValue": 1
		},
```